node-runner-cli/radixnode.py

```diff
@@ -222,7 +222,8 @@ def vote(args):
             "by performing this action, the validator will record support for this fork onto the ledger.\n" +
             f"If you choose to downgrade the software to no longer support this fork, you should manually remove the validator's support for the candidate fork with the withdraw vote action.{bcolors.ENDC}"
         )
-    core_api_helper.vote(True)
+        core_api_helper.vote(True)
+    else: print(f"{bcolors.WARNING}No vote is required, therefore no voting action will be performed.{bcolors.ENDC}")
```

Comment:
I think it's more idiomatic python to require a new line (and appropriate indentation) for any new block - I didn't even know this was possible tbh :D